lib/transaction.d.ts

```diff
@@ -1,25 +1,50 @@
 import { Logging } from './model'
 import { Promise } from './promise'
-import { Sequelize } from './sequelize'
-
-/**
- * The transaction object is used to identify a running transaction. It is created by calling
- * `Sequelize.transaction()`.
- *
- * To run a query under a transaction, you should pass the transaction in the options object.
- */
-export class Transaction {
-    constructor(sequelize: Sequelize, options: TransactionOptions)
+import { Deferrable, Sequelize } from './sequelize'
 
+// tslint:disable-next-line
```

Comment:
please only disable specific rules and mention why they need to be disabled. Maybe the rules shouldn't be enabled in the first place.